### PR TITLE
waec sample

### PR DESCRIPTION
**Pull request by samuelIkoli**
Following this issue https://github.com/Udayraj123/OMRChecker/issues/40 , I added Sample image from the West African Examinaton Council (WAEC) and it's template.

### CODE REVIEW

File: samples/community/samuelIkoli/template.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "q1",
+        "q2",
+        "q3",
+        "q4",
+        "q5",
+        "q6",
+        "q7",
+        "q8",
+        "q9",
+        "q10"
```
Comment (Udayraj123) [inline samples/community/samuelIkoli/template.json:28]: looks like the actual OMR image has more questions available. Can you configure the complete template for all questions visible? Also if possible add one more image(of same layout) to test your template with.
 
Also you can use the short syntax "q1..10", etc for consecutive questions.

**Review by Udayraj123 — CHANGES_REQUESTED** [review]
Hey @samuelIkoli, thanks for the PR! Can you please update the image correctly? It looks like it's not scanned to cover the full page

File: samples/community/samuelIkoli/template.json [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        "q15",
+        "q16",
+        "q17",
+        "q18",
+        "q19",
+        "q20"
+      ],
+      "bubblesGap": 33,
+      "labelsGap": 20
+    },"MCQBlock2": {
```
Comment (Udayraj123) [inline samples/community/samuelIkoli/template.json:42]: Can you please prettify the json file as well?

File: samples/community/samuelIkoli/waec_sample.jpeg
Comment (Udayraj123) [inline samples/community/samuelIkoli/waec_sample.jpeg:1]: This image seems to be cropped manually, that's fine as this is a community sample.
Can you just compress this image and submit again?

**Review by Udayraj123 — CHANGES_REQUESTED** [review]

File: samples/community/samuelIkoli/template.json (hunk 2 above)
Comment (samuelIkoli) [reply samples/community/samuelIkoli/template.json]: Done

File: samples/community/samuelIkoli/waec_sample.jpeg
Comment (samuelIkoli) [reply samples/community/samuelIkoli/waec_sample.jpeg:1]: Done

**Review by Udayraj123 — APPROVED** [review]

**Comment by Udayraj123** [comment]
Thanks for a new sample @samuelIkoli!

**Pull request merged**